README.md

```diff
@@ -0,0 +1,46 @@
+# enterprise-catalog, license-manager, enterprise-access and enterprise-subsidy plugin for [Tutor](https://docs.tutor.overhang.io)
+
+## Installation
+
+```sh
+pip install git+https://github.com/open-craft/tutor-contrib-enterprisecatalog
+```
+
+## Usage
+
+```sh
+# This plugin depends on discovery, ecommerce and mfe tutor plugins
+tutor plugins enable enterprise-catalog discovery ecommerce mfe
+# Build required images
+tutor images build enterprise-catalog enterprise-catalog-worker license-manager \
```

Comment:
This one is fine. The frontend repo (fixed in https://github.com/overhangio/tutor-ecommerce/commit/0a619ca3701244bd367741aa6459d2cded14a09d) was affected.

We should add a warning about it to the README because it's currently impossible to set this up by simply following the instructions.